spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -312,6 +313,23 @@ object SpdxDeclaredLicenseMapping {
         "the Apache License, ASL Version 2.0" to APACHE_2_0.toExpression()
     )
 
+    init {
```

Comment:
I would prefer to have both sanity checks as unit tests instead, there's no need to execute them on every start of ORT.

Reply:
Good idea, will do.